docs/Using-Pantheon/Transactions/Revert-Reason.md

```diff
@@ -0,0 +1,75 @@
+description: Including revert reason in transactions 
+<!--- END of page meta data -->
+
+# Revert Reason 
+
+In contracts, the [`revert`](https://solidity.readthedocs.io/en/v0.5.10/control-structures.html#revert) function 
+triggers an exception to flag an error and revert the current call. An optional string message containing 
+information about the error is passed back to the caller.
```

Comment:
returned to the client from the EVM (not to the caller per se).

Reply:
Fixed